lib/capybara/selenium/node.rb

```diff
@@ -33,14 +33,15 @@ def value
   # Set the value of the form element to the given value.
   #
   # @param [String] value    The new value
-  # @param [Hash{}] options  Driver specific options for how to set the value
+  # @param [Hash{}] options  Driver specific options for how to set the value. Take default values from {Capybara#default_set_options}
   # @option options [Symbol,Array] :clear (nil) The method used to clear the previous value <br/>
   #   nil => clear via javascript <br/>
   #   :none =>  append the new value to the existing value <br/>
   #   :backspace => send backspace keystrokes to clear the field <br/>
   #   Array => an array of keys to send before the value being set, e.g. [[:command, 'a'], :backspace]
   def set(value, **options)
     raise ArgumentError, "Value cannot be an Array when 'multiple' attribute is not present. Not a #{value.class}" if value.is_a?(Array) && !multiple?
+    options = Capybara.default_set_options.merge(options)
```

Comment:
This should be implemented in capybara/node/element.rb so it affects all drivers, not in selenium specific code.  Also it should pull from `session_options.default_set_options` not `Capybara_default_set_options` so it works in "threadsafe" mode too - see https://github.com/teamcapybara/capybara/blob/master/spec/per_session_config_spec.rb